jobrunner/config.py

```diff
@@ -104,6 +104,9 @@ class ConfigException(Exception):
 )
 
 
+ALLOWED_GITHUB_ORGS = os.environ.get("ALLOWED_GITHUB_ORGS", "opensafely").split()
```

Comment:
Would splitting on comma cause less potential confusion here?  It's what a few libraries do for "list" env vars, and IIRC shells typically treat space as a delimiter so could cause issues there?

Reply:
Yes, good point.